src/main/resources/application.conf

```diff
@@ -3,27 +3,21 @@ database {
   config = hsqldb
 
   hsqldb {
-    url = "jdbc:hsqldb:mem:${slick.uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
-    driver = "org.hsqldb.jdbcDriver"
-    slick.driver = "slick.driver.HsqldbDriver"
+    db.url = "jdbc:hsqldb:mem:memdb;shutdown=false;hsqldb.tx=mvcc"
+    db.driver = "org.hsqldb.jdbcDriver"
```

Comment:
It is JDBC info, mixed in with other slick/hikari specific info. Will block it.

But [slick](http://slick.typesafe.com/doc/3.1.0/database.html?highlight=db#databaseconfig) [wants](https://github.com/slick/slick/blob/3.1.0/slick/src/main/scala/slick/backend/DatabaseConfig.scala#L73) the path named "db", so it shall be.